modules/tinymce/src/themes/silver/main/ts/ui/dialog/UrlInput.ts

```diff
@@ -119,14 +119,20 @@ export const renderUrlInput = (
         disabled: () => !spec.enabled || providersBackstage.isDisabled()
       }),
       Tabstopping.config({}),
-      AddEventsBehaviour.config('urlinput-events', Arr.flatten([
+      AddEventsBehaviour.config('urlinput-events',
         // We want to get fast feedback for the link dialog, but not sure about others
-        spec.filetype === 'file' ? [
-          AlloyEvents.run(NativeEvents.input(), (comp) => {
-            AlloyTriggers.emitWith(comp, formChangeEvent, { name: spec.name });
-          })
-        ] : [ ],
         [
+          AlloyEvents.run(NativeEvents.input(), (comp) => {
+            const currentValue = Value.get(comp.element);
+            const trimmedValue = currentValue.trim();
```

Comment:
Should we be using Strings.trim from katamari on this or do we not care I guess it's so small and not that common that minification is less of a concern?

Reply:
I'm not sure `Strings.trim` is really the best solution anymore. It's using regex? And was probably created when we had to support IE8. It might be _slightly_ smaller when minified, but gzip erases the difference.

Reply:
Right yeah there are probably weird unicode white space characters the native trim is better of handling than say a regexp.